src/games/GameDialog.tsx

```diff
@@ -530,7 +530,7 @@ const GameDialogActions: React.FC<{
                 continue
               }
 
-              if (fileEntry.isIwad) {
+              if (fileEntry.isIwad && !iwad) {
```

Comment:
This is also a good reason to switch how the iwad is specified as in https://github.com/mikew/wadpunk/issues/44